src/theme/material/calendar.m.css

```diff
@@ -41,6 +43,15 @@
 	text-align: center;
 	position: relative;
 	z-index: 0;
+	cursor: pointer;
+	display: inline-flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.monthFields,
+.yearFields {
+	border: none;
```

Comment:
Padding and margin should be set to 0 here too I believe. The years were 4 to a row when the resets were applied